Добавлен календарь.

Aleksandrln opened the pull request:
Висит в левом меню.
Выглядит вот так: 
![image](https://cloud.githubusercontent.com/assets/24756493/21878021/4ef6f392-d8a0-11e6-8708-4cc9cdeec86a.png)
Слегка выбивается из дизайна, знатоки css смогут легко это  поправить.
При наведении на дату, выскакивает всплывающее окошко. Выглядит не очень, но опять же поправимо. 
![image](https://cloud.githubusercontent.com/assets/24756493/21878070/a743b04e-d8a0-11e6-909a-86101fa8037d.png)
Календарь отображает дни рождения и события на дату. Праздники не отображает.
У всплывающего окошка есть баг. Иногда (когда событий много) оно закрывает собой весь календарь.

Aleksandrln committed:
Добавлен календарь.

EgoAkk commented:
Может лучше сделать всплывающее окно по нажатию на дату? А то с наведением будет много ложных срабатываний, когда мышкой пробегаешь по экрану

Aleksandrln commented:
Это смотря с какой скоростью пробегать)
Как вариант, могу сделать выбор в настройках click or mouseover
![safegif](https://cloud.githubusercontent.com/assets/24756493/21879078/6ef6087a-d8a7-11e6-9b8c-2cec90c040c0.gif)

Aleksandrln committed:
Исправлен баг.  При клике на заголовок и попытке выбрать конкретный месяц, календарь очищался.

Aleksandrln committed:
Теперь календарь отображается на всех страницах.

Aleksandrln committed:
Добавлены опции:
1) не показывать прошедшие события (в календаре отображаются только события, которые должны произойти)
2) всплывающая подсказка по клику, а не при наведении ( для появления всплывающей подсказки необходимо кликнуть на дату)

Saiv46 reviewed (approved):
LGTM, неплохо бы замержить
